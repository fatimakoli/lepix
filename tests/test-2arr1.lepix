fun main(): int
{
	var arr : int[[4,3]];

	var i:int;
	var j:int;
	for(i = 0; i<4; i=i+1)
	{
		for(j = 0; j<3; j=j+1)
		{
			print(88888);
			print(i);
			print(j);
			print(i+j);
			arr[i,j] = i+j;
			print(arr[i,j]);
		}
	}
	return 0;
}