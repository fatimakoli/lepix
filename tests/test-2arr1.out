88888
0
0
0
0
88888
0
1
1
1
88888
0
2
2
2
88888
1
0
1
1
88888
1
1
2
2
88888
1
2
3
3
88888
2
0
2
2
88888
2
1
3
3
88888
2
2
4
4
88888
3
0
3
3
88888
3
1
4
4
88888
3
2
5
5
